src/sass/edu-benefits.scss

```diff
@@ -25,3 +25,13 @@ select {
 legend {
   @extend h4;
 }
+
+.form-indent {
```

Comment:
@akainic Are there existing classes that I can use instead of these? One is to indent some text and the other is to create a box with a gray border around it for the checkboxes that show up if you choose chapter 33.